ngraph/CMakeLists.txt

```diff
@@ -306,20 +313,25 @@ if (NGRAPH_ONNX_IMPORT_ENABLE)
 
     set(BEFORE_ONNX_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
     set(BUILD_SHARED_LIBS ON)
+    set(BUILD_STANDALONE_STATIC OFF)
 
     if (NOT NGRAPH_USE_SYSTEM_PROTOBUF)
```

Comment:
have we checked that system protobuf has static libraries?

Reply:
System protobuf has static libraries, but these are not used in a project. There is an option "NGRAPH_USE_SYSTEM_PROTOBUF", but it is always 'off' and enabling this produces ONNX build errors. So, I guess it is not used in any builds and probably should be removed under some cleanup activity. But I'd prefer to not do this in this PR